modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/Helpers/JsonFromTokenUri.swift

```diff
@@ -35,7 +35,8 @@ final class JsonFromTokenUri {
             blockchainProvider: blockchainProvider,
             uriMapper: TokenUriMapper(hostMappers: [
                 HostBasedTokenUriMapper(host: "api.mintkudos.xyz"),
-                HostBasedTokenUriMapper(host: "api.walletads.io")
+                HostBasedTokenUriMapper(host: "api.walletads.io"),
```

Comment:
How do we get the original URL that has to be converted? eg. `https://api.mintkudos.xyz/metadata/3ba0000000000000000000000000000000000000000000000000000000000000` ?